mllib/src/main/scala/org/apache/spark/ml/classification/DecisionTreeClassifier.scala

```diff
@@ -81,7 +81,7 @@ class DecisionTreeClassifier @Since("1.4.0") (
    * E.g. 10 means that the cache will get checkpointed every 10 iterations.
    * This is only used if cacheNodeIds is true and if the checkpoint directory is set in
    * [[org.apache.spark.SparkContext]].
-   * Must be >= 1.
+   * Must be {@literal >= 1}.
```

Comment:
Yes, I read about that but figured that the literal text is probably meant to be a logical part of code. Which, in this case, is `>= 1`.

Making just `>` literal would make sense if you want to minimize the cosmetic differences between the characters. However, as I did not know whether the resulting code block would retain the same styling in the future, I opted for logical separation. If scaladoc later changed the styling of literals to, for example, the GitHub style for code, the result would look very bad. `>= 1` would still look relatively good.

But if the `{@literal >}` style is widely used in the Spark project, I can change it.